cmd/scollector/collectors/dell_hw.go

```diff
@@ -93,16 +93,24 @@ func c_omreport_ps() (opentsdb.MultiDataPoint, error) {
 func c_omreport_ps_amps() (opentsdb.MultiDataPoint, error) {
 	var md opentsdb.MultiDataPoint
 	readOmreport(func(fields []string) {
-		if len(fields) != 2 || !strings.Contains(fields[0], "Current") {
+		if (len(fields) != 2 || !strings.Contains(fields[0], "Current")) && !(len(fields) == 6 && fields[2] == "System Board Pwr Consumption") {
 			return
 		}
-		i_fields := strings.Split(fields[0], "Current")
-		v_fields := strings.Fields(fields[1])
-		if len(i_fields) < 2 && len(v_fields) < 2 {
-			return
+		if len(fields) == 2 {
+			i_fields := strings.Split(fields[0], "Current")
+			v_fields := strings.Fields(fields[1])
+			if len(i_fields) < 2 && len(v_fields) < 2 {
+				return
+			}
+			id := strings.Replace(i_fields[0], " ", "", -1)
+			Add(&md, "hw.chassis.current.reading", v_fields[0], opentsdb.TagSet{"id": id}, metadata.Gauge, metadata.A, descDellHWCurrent)
+		} else {
+			v_fields := strings.Fields(fields[3])
```

Comment:
Not sure what you mean. There are 2 fields for the Current and 6 for the Power. This is what we are parsing:

PS C:\windows\system32> omreport chassis pwrmonitoring -fmt ssv
Power Consumption Information

Power Consumption

Index;Status;Probe Name;Reading;Warning Threshold;Failure Threshold
2;Ok;System Board Pwr Consumption;210 W;1792 W;1974 W

Amperage
PS1 Current 1;0.4 A
PS2 Current 2;0.6 A